src/dune_rules/lib_file_deps.ml

```diff
@@ -1,22 +1,31 @@
 open Import
 
 module Group = struct
-  type t =
+  type ocaml =
     | Cmi
     | Cmx
+
+  type melange = Melange.Cm_kind.t
```

Comment:
We don't need this alias I think.

Reply:
https://github.com/jchavarri/dune/pull/4/commits/a4c3b5661a8f479b58b2585f8a4e4248ba1d3110